README.md

```diff
@@ -88,18 +138,28 @@ L.tileLayer('https://{s}.tiles.mapbox.com/v3/anishshah101.ipm9j6em/{z}/{x}/{y}.p
   id: 'examples.map-i86knfo3'
 }).addTo(map);
 
+// keymap guide
+var keymapper = {
+  position: "topright",
+  el: "div",
+  styleClass: "l-container",
+  DOMString:
+    "<b><center><h3>Keymappings</h3><center><hr/></center></center><ul><li>L: Lock overlay</li><li>O: Outline overlay</li><li>R: Rotate overlay</li><li>RR: Distort overlay</li><li>T: Transparent overlay&nbsp;&nbsp;&nbsp;&nbsp;</li><li>DEL: Remove overlay</li></ul></b>"
+};
+
 // create an image
-img = new L.DistortableImageOverlay(
-  'example.png', {
-    corners: [
-      new L.latLng(51.52,-0.10),
-      new L.latLng(51.52,-0.14),
-      new L.latLng(51.50,-0.10),
-      new L.latLng(51.50,-0.14)
-    ],
-    fullResolutionSrc: 'large.jpg' // optionally pass in a higher resolution image to use in full-res exporting
-  }
-).addTo(map);
+img = new L.DistortableImageOverlay("example.png", {
+  suppressToolbar: false,
+  customToolbarArray: [keymapper],
+  hidden_tools: [ToggleEditable, ToggleExport, RemoveOverlay],

```

Comment:
This is really cool - would it make sense for the hidden-ness to be a property of the tool, or even of the toolbar, rather than a property of the ImageOverlay? Could we achieve that?